docs/Reference/EthSigner-CLI.md

```diff
@@ -52,20 +74,18 @@ Timeout period (in milliseconds) for downstream requests. Default is 5000.
 ```
 
 ```bash tab="Example"
---downstream-http-request-timeout=3000
+--downstream-http-request-timeout=10
```

Comment:
Now that we have standardised on milliseconds thinking this example should be a larger number like it was before

Reply:
Fixed